hoomd/md/methods/__init__.py

```diff
@@ -15,6 +15,21 @@
 .. rubric:: Integration methods with constraints
 
 For methods that constrain motion to a manifold see `hoomd.md.methods.rattle`.
+
+.. rubric:: Preparation
```

Comment:
I am not sure that preparation makes sense as a title from a documentation reader's perspective. Maybe "Attaching an Integrator" could be the title.